tracer/src/Datadog.Trace/Configuration/ImmutableExporterSettings.cs

```diff
@@ -90,14 +94,25 @@ public ImmutableExporterSettings(ExporterSettings settings)
         /// </summary>
         public int PartialFlushMinSpans { get; }
 
+        /// <summary>
+        /// Gets the unix domain socket path where the Tracer can connect to the Agent.
+        /// </summary>
+        /// <seealso cref="ConfigurationKeys.TracesUnixDomainSocketPath"/>
+        public string TracesUnixDomainSocketPath { get; }
+
+        /// <summary>
+        /// Gets the unix domain socket path where the Tracer can send stats.
+        /// </summary>
+        /// <seealso cref="ConfigurationKeys.MetricsUnixDomainSocketPath"/>
+        public string MetricsUnixDomainSocketPath { get; }
+
         /// <summary>
         /// Gets the transport used to send traces to the Agent.
         /// </summary>
         internal TracesTransportType TracesTransport { get; }
 
         /// <summary>
         /// Gets the transport used to connect to the DogStatsD.
-        /// Default is <c>TransportStrategy.Tcp</c>.
```

Comment:
`/// Default is <c>GodOnlyKnows</c>`

Reply:
Well this comment was very wrong anyways.